packages/sdk-react/src/mocks/paymentReminders/reminderFixtures.ts

```diff
@@ -0,0 +1,102 @@
+import { paths } from '@/api';
+import { entityIds } from '@/mocks/entities';
+import { faker } from '@faker-js/faker';
+
+const sendPaymentNameGenerator = (days: number, type: string) =>
+  `Send ${type} reminders ${days} days before due date`;
+
+export const paymentReminderListFixture: paths['/payment_reminders']['get']['responses']['200']['content']['application/json']['data'] =
+  new Array(5).fill('_').map(() => ({
+    id: faker.string.nanoid(),
+    name: sendPaymentNameGenerator(
+      faker.datatype.number({ min: 1, max: 30 }),
+      'payment'
+    ),
+    created_at: faker.date.past().toString(),
+    updated_at: faker.date.past().toString(),
+    entity_id: entityIds[0],
+    status: faker.datatype.boolean() ? 'active' : 'deleted',
+  }));
+
+export const overdueReminderListFixture: paths['/overdue_reminders']['get']['responses']['200']['content']['application/json']['data'] =
+  new Array(5).fill('_').map(() => ({
+    id: faker.string.nanoid(),
+    name: sendPaymentNameGenerator(
+      faker.datatype.number({ min: 1, max: 30 }),
+      'overdue'
+    ),
+    created_at: faker.date.past().toString(),
+    updated_at: faker.date.past().toString(),
+  }));
+
+export type OverdueReminder =
```

Comment:
By default, we are not exporting anything until it's needed. This one does not need to be exported either.